addons/mail/static/src/js/thread.js

```diff
@@ -34,7 +34,8 @@ var Thread = Widget.extend({
             var message_id = $(event.currentTarget).data('message-id');
             this.trigger("mark_as_read", message_id);
         },
-        "click .o_thread_message_star": function (event) {
+        "click .o_thread_message_star:not(.disabled)": function (event) {
+            $(event.currentTarget).addClass('disabled');
```

Comment:
@ged-odoo ,
We add this line, because If user clicks multiple times quickly on star button in lower connection it show wrong value.
https://drive.google.com/file/d/0B8yjRSZ_xdLMRDZRRHRWZFlRdEU/view?usp=sharing